apps/ios/package.json

```diff
@@ -16,13 +16,13 @@
     "test": "fluentui-scripts jest",
     "bundle": "fluentui-scripts metro --cli",
     "bundle-dev": "fluentui-scripts metro --dev --cli",
-    "ios": "react-native run-ios --scheme ReactTestApp --project-path src"
+    "ios": "react-native run-ios --project-path src"
   },
   "dependencies": {
     "@fluentui-react-native/tester": "^0.13.7",
     "react": "16.11.0",
-    "react-native": "0.62.2",
-    "react-native-test-app": "^0.2.1"
+    "react-native": "^0.62.2",
```

Comment:
I don't think we should use the caret here. Although I used to think these version carets were handy, there's something to be said about having an easy way to tell the exact dependency version we're on. It's a bit more overhead to update it when we need to, but that consistency in reproducibility is really nice if ever there's a problem.